resources/lib/process.py

```diff
@@ -259,6 +259,8 @@ def start_info_actions(info, params):
     elif info == "openinfodialog":
         if xbmc.getCondVisibility("System.HasActiveModalDialog"):
```

Comment:
just make this `if xbmc.getCondVisibility("System.HasActiveModalDialog | System.HasModalDialog")`